site/en/blog/show-picker/index.md

```diff
@@ -0,0 +1,203 @@
+  <figcaption class="w-figcaption">Browser date pickers in Chrome Desktop, Chrome Mobile, Safari Desktop, Safari Mobile, and Firefox Desktop (July 2021).</figcaption>
+</figure>
+
+{% Aside %}
+Unlike a third-party picker widget, a browser picker is familiar to the user,
+has great specific platform support, and is always maintained as part of the
+browser.
+{% endAside %}
+
+## How to show a picker {: #how-to }
+
+Calling `showPicker()` on an `<input>` element returns a promise that resolves
+when a browser picker is shown to the user. It must be called in response to a
+user gesture such as a touch gesture or mouse click; otherwise it will reject.
+For security reasons,it will also reject when it's called in a cross-origin
+iframe.
+
+A browser picker is shown when the `<input>` element is one of these types:
+`"date"`, `"month"`, `"week"`, `"time"`, `"datetime-local"`, `"color"`, and
+`"file"`.
+
+The example below shows you how to open a browser date picker.
+
+```html/9
+<input type="date">
+<button>Show the date picker</button>
+
+<script>
+  const button = document.querySelector("button");
+  const dateInput = document.querySelector("input");
+
+  button.addEventListener("click", async () => {
+    try {
+      await dateInput.showPicker();
+      // A date picker is shown.
+    } catch (error) {
+      // Use external library when this fails.
+    }
+  });
+</script>
+```
+
+A browser picker may also be shown if the `<input>` element has some suggestions
+powered by `<datalist>` or `"autocomplete"`.
+
+The example below shows you how to open a browser picker with `<datalist>`.
+
+```html/16
+<datalist id="ice-cream-flavors">
+  <option value="Chocolate"> </option>
+  <option value="Coconut"> </option>
+  <option value="Mint"> </option>
+  <option value="Strawberry"> </option>
+  <option value="Vanilla"> </option>
+</datalist>
+<input type="text" list="ice-cream-flavors">
+<button>Show the suggestions</button>
+
+<script>
+  const button = document.querySelector("button");
+  const iceCreamFlavorsInput = document.querySelector("input");
+
+  button.addEventListener("click", async () => {
+    try {
+      await iceCreamFlavorsInput.showPicker();
+      // A picker containing some ice cream flavors is shown.
+    } catch (error) {
+      // Use external library when this fails.
+    }
+  });
+</script>
+```
+
+## Demo
+
+A playground demo is available at
+[https://show-picker.glitch.me/demo.html][demo] for you to play with all
+supported pickers by the browser.
+
+{% Video src="video/vvhSqZboQoZZN9wBvoXq72wzGAf1/MMPmrnJnMvFX3eMCUWK9.mov",
+autoplay="true", muted="true", loop="true", class="screenshot" %}
+
+## Browser support
+
+At the time of writing, `showPicker()` is available in Chrome&nbsp;99.
+
+## What's next {: #future }
+
+At the time of writing, `showPicker()` is new to the web platform. Some
+additional work may be needed in the future:
```

Comment:
```suggestion
At the time of writing, `showPicker()` is new to the web platform. You
may need to do additional work in the future:
```

Reply:
Passive to active voice.

Reply:
If I understand correctly, that's not the same.
You're saying reader may need to **do** some work. Nope. They may need some additional work from the web platform folks to be able to get what they want.

Reply:
Thus demonstrating one of the problems with passive voice. Just change the subject from 'You' to something indicating the actors, maybe "browser vendoprs".

Reply:
Browser vendors, WhatWG html spec folks, and web developers in general interested in contributing to the web platform. It's hard to pick one.

Reply:
I just changed the suggestion. PTAL.